### PR TITLE
comited by sivaprasad kalyan for testing pull request cleared 1

### PR DESCRIPTION
**Pull request by siddharth-java-204**
this comment for checking pull request
this comment created by siddharth

### CODE REVIEW

**Comment by kalyanshiva12** [comment]
hi this is pull request for oneable

**Comment by kalyanshiva12** [comment]
cghghfgh

**Comment by kalyanshiva12** [comment]
jkjkk

**Comment by kalyanshiva12** [comment]
bbmnbnmbmbjhbhjbmnbhj

**Comment by kalyanshiva12** [comment]
ggggggggggggggggggggggggggg

**Comment by kalyanshiva12** [comment]
shiva

**Comment by kalyanshiva12** [comment]
vgghvgvbhjb

File: shiva1.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+hi i am shiva 
+i am working in oneable
```
Comment (siddharth-java-204) [inline shiva1.txt:2]: here i did wrong

File: shiva1.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+hi i am shiva 
+i am working in oneable
+hi i am shiva
```
Comment (siddharth-java-204) [inline shiva1.txt:5]: this is good

File: shiva1.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+hi i am shiva 
+i am working in oneable
+hi i am shiva 
+i am working in oneable
+hi i am shiva 
+i am working in oneable
+hi i am shiva
```
Comment (siddharth-java-204) [inline shiva1.txt:7]: hi siva

**Review by siddharth-java-204 — COMMENTED** [review]
this all you hava to check

File: shiva1.txt (hunk 1 above)
Comment (siddharth-java-204) [reply shiva1.txt:2]: checkout

File: shiva1.txt (hunk 3 above)
Comment (siddharth-java-204) [reply shiva1.txt:7]: checkout

Comment (siddharth-java-204) [reply shiva1.txt:7]: kfdhfkjhdkfdfd

Comment (siddharth-java-204) [reply shiva1.txt:7]: jfdkjhfjkdhkgh

**Review by siddharth-java-204 — COMMENTED** [review]
approve
this id approved

File: shiva1.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (siddharth-java-204) [inline shiva1.txt:1]: approved

**Pull request merged**